src/AdminConsole/Pages/Index.cshtml.cs

```diff
@@ -10,7 +10,12 @@ public async Task<IActionResult> OnGetAsync()
     {
         if (await setupService.HasSetupCompletedAsync())
         {
-            return RedirectToPage(HttpContext.User.Identity!.IsAuthenticated ? "/account/login" : "/Organization/overview");
+            if (HttpContext.User.Identity!.IsAuthenticated)
+            {
+                return RedirectToPage("/account/login");
+            }
+
+            return Redirect("/organization/overview");
```

Comment:
RedirectToPage(...) would work if `/organization/overview' was a Razor page, but given it doesn't exist it crashes. The `Redirect()` one works for Blazor. It wasn't spotted before because of the ternary expression.

Reply:
The `Organization/overview` page was migrated to Blazor, hence the change.